### DISCUSSION TITLE
경쟁에 대한 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#91] [Q&A] [answered]
멘토님..! 개인적인 질문인데 멘토님의 생각이 궁금해서 질문드립니다.  

멘토님은 경쟁에 대해서 어떻게 생각하시는지 궁금합니다

저는 나름 성장하기 좋은 상황이라고 생각하기도 하고 때로는 자신만의 경쟁, 열등감이 조금이라도 있어야 한다고 생각합니다.

*혼자 생각하다가 멘토님의 생각이 궁금하여 질문한거라 편하게 답변해주시거나 넘기셔도 됩니다..!*

**Comment by jongfeel** [comment, marked as answer]
선의의 경쟁이라면 좋을 수 있습니다.
특히 비슷한 기술이나 사업 목표를 가진 회사간의 경쟁은 더 그렇죠.
경쟁에서 우위를 점하는 회사는 더 많은 이익을 가져갈 수 있고
그렇지 못한 회사는 경쟁 우위를 점하기 위해 더 많은 노력과 시간을 들이게 되니까요.

만약 소프트웨어 개발에서의 경쟁이라면 협력으로 바꿔야 합니다.
소프트웨어 개발 할 때는 경쟁이 큰 의미는 없습니다.
경쟁의 본질적인 특징으로 모두가 만족스럽거나 원하는 결과를 얻지 못한다는 부분이 있기 때문에,
함께 협력해서 만들어 나가야 하는 소프트웨어 개발하는 쪽에서는 오히려 좋지 않다고 봅니다.

**Reply by fkdl0048** [reply]
정말 시각이 다르네요..

좀 더 큰 회사나 단체같이 사회정도의 경쟁은 좋은 순환구조로 볼 수 있지만  

소프트웨어 개발 측면에서 경쟁 자체는 크게 의미가 없군요.. 확실히 오픈소스나 기여하는 방식이나 정보의 공유가 매우 빠른것도 큰 틀에서 보면 비슷한 맥락일 것 같습니다.

좋은 답변 감사합니다..!
